workos/src/main/kotlin/com/workos/common/exceptions/UnprocessableEntityException.kt

```diff
@@ -2,6 +2,10 @@ package com.workos.common.exceptions
 
 import com.workos.common.http.UnprocessableEntityExceptionResponse.EntityError
 
-class UnprocessableEntityException(val errors: List<EntityError>?, val requestId: String) : Exception("UnprocessableEntityException") {
```

Comment:
This Java app would just show "UnprocessableEntityException" rather than the error message from the REST API